src/main/java/net/countercraft/movecraft/combat/features/tracking/DamageTracking.java

```diff
@@ -10,22 +10,21 @@
 import net.countercraft.movecraft.events.CraftSinkEvent;
 import org.bukkit.Bukkit;
 import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Wildcard imports can increase compile times and lead to unexpected behavior, as such we avoid using them in Movecraft and its addons.  By default IntelliJ IDEA will use them, so you will want to go into Settings > Editor > Code Style > Java > Imports and set `"Class count to use import with '*'` and `Names count to use static import with '*'` to `9999`.

Reply:
Fixed